ttg/ttg/parsec/ttg.h

```diff
@@ -161,7 +164,12 @@ namespace ttg_parsec {
 
    public:
     static constexpr const int PARSEC_TTG_MAX_AM_SIZE = 1024 * 1024;
-    WorldImpl(int *argc, char **argv[], int ncores) : WorldImplBase(query_comm_size(), query_comm_rank()) {
+    WorldImpl(int *argc, char **argv[], int ncores) : WorldImplBase(query_comm_size(), query_comm_rank())
+#if defined(PARSEC_PROF_TRACE) 
```

Comment:
I would make that initialization independent of `PARSEC_PROF_TRACE` but make it an assignment here https://github.com/TESSEorg/ttg/pull/227/files#diff-b16b2b248d6db2366353f63da94be820a8905343d7452acc0639ffcaf3100068R330